### ISSUE TITLE
subscribe.wordpress.com: unclear wording

### ISSUE THREAD

**Issue by ellatrix** [Automattic/wp-calypso#56691]
### Quick summary

When I manage my **email** subscription, I see a list of blogs that I'm following with an unfollow button next to them. It's not clear to me if that button unfollows the blog on wordpress.com or if it unsubscribes me from the emails. (I just want to unsubscribe from emails but still follow the blog.)

As a workaround, I went to settings and tried "Block all email updates from blogs you’re following on WordPress.com". Here it's also unclear if that includes @ mentions or just new posts. Would be good to have some granularity in this setting.

I went on to just try to unfollow, and it turns out that it indeed only removes the email subscription.

Reading the description at the top is also a bit confusing:

"This is the management page for following blogs. You will receive an email at ... whenever a new post is made on these sites. Unfollow a blog to stop receiving emails from that blog. Go to Settings to stop receiving all emails from WordPress.com."

* "for following blogs": Doesn't explain it is just for email subscription. It implies I manage following blogs in general.
* "Unfollow a blog to stop receiving emails from that blog.": Implies I will unfollow the blog AND logically will stop receiving emails. But in fact, I will NOT unfollow the blog, it will merely stop the email subscription if I press **unfollow**.

In summary, the use of **unfollow** is ambiguous and it's confusing. I would suggest using "unsubscribe" or even clearer "unsubscribe from email".

### Steps to reproduce

Ensure you have some email subscriptions and go to subscribe.wordpress.com.

### What you expected to happen

See above

### What actually happened

See above

### Context

Personal use

**Comment by cuemarie** [comment]
_This report was found during a recent backlog sweep._

📌 FINDINGS/SCREENSHOTS/VIDEO
- Subscription management has been revamped:

![Markup on 2023-07-18 at 16:10:12](https://github.com/Automattic/wp-calypso/assets/27249804/41337312-d68a-485b-b065-0d83064c8892)

📌 ACTIONS
- Closed as not reproducible / no longer relevant
